### ISSUE TITLE
Add dedicated wrapper repos to phetmarks

### ISSUE THREAD

**Issue by zepumph** [phetsims/phetmarks#34]
This would be pretty straightforward to read from package.phet.phet-io.wrappers if there was a convention for what the runnable was called and where it was located in each dedicated repo. @samreid @chrisklus do you think it is worth the effort to make this consistent?

Then classroom-activity could be in only FAMB, and hookes-law-energy could be only in Hookes law.

**Comment by samreid** [comment]
This seems to be a nice enhancement, would be good for long term to make it easier for everyone to use.  Medium priority?

**Comment by zepumph** [comment]
Sure. What do you think our convention for filenaming should be. Top level {{wrapper}}.html where wrapper is the repo but doesn't include "phet-io-wrapper-"?

**Comment by samreid** [comment]
Are they currently named `phet-io-wrapper-{{name}}/index.html`?  What all will have to change?

**Comment by zepumph** [comment]
Yes yes that's right. I think that's true and good. The haptics wrapper doesn't have that though, and could use a top level index page pointing to its runnables. Let's say every dedicated wrapper repo needs a top level index page.

**Comment by zepumph** [comment]
At this point, the number of dedicated wrapper repos is really really tiny. Phetmarks is totally not set up to make a dynamic ajax for each phet-io sim's package.json to see what extra wrappers it may need.  I would much prefer to close this issue than to spend energy on it. It is annoying for things like the lab-book for BLL, but it also isn't worth the rewrite of phetmarks, or the additional data/ list that holds what sims need dedicated wrappers. I also think it would be more confusing than helpful if we made classroom-activity and lab-book available to all phet-io sims (with mostly broken links).

Closing
